freqtrade/freqai/RL/BaseReinforcementLearningModel.py

```diff
@@ -398,3 +401,51 @@ def _init() -> gym.Env:
         return env
     set_random_seed(seed)
     return _init
+
+
+class TensorboardCallback(BaseCallback):
```

Comment:
Might be best to move this to it's own file - keeping multiple classes in a single file starts to get disorganized quickly.